frontends/packages/livnote/desktop/components/notes/notes.ts

```
@@ -851,6 +863,90 @@ export class Notes {
+
+  /**
+   * Update comment thread position (for document changes)
+   */
+  updateCommentThreadPosition(threadId: string, newPosition: CommentPosition): boolean {
+    return this.commentsService.updateThreadPosition(threadId, newPosition);
+  }
+
+  /**
+   * Subscribe to comment events
+   */
+  onCommentUpdate(eventType: string, callback: Function): void {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)